dist/windows/qt.conf

```diff
@@ -2,4 +2,4 @@
 Translations = translations
 
 [Platforms]
-WindowsArguments = dpiawareness=1
+WindowsArguments = dpiawareness=0
```

Comment:
>Is this due to Qt 5.6 being released?

Yes, Qt 5.6 work differently.

>If yes, should we remove it altogether?

I think no. If remove it, the default scaling `dpiawareness=2` is worse than current.

-

This is complicated, users have different opinions about this:
* Myself thinks `0` is better:
 Qt renders at 96 dpi (the native for non-hidpi screen), then upscales to screen's dpi, high dpi screen will see it blury.
 **The point is: all dialogs' width & height are the same & correct as we designed it**

* [One user](https://github.com/qbittorrent/qBittorrent/issues/2963#issuecomment-204056120) find `1` is better:
 Qt renders at some system default DPI, then upscale/downscale when needed.
 Although it gives crisp font & UI on hidpi screen, the width & height are messed up and fixing this is troublesome (I don't think this should be the way).

In short, I've been thinking turning this into an option under "advanced option page", only problem is: the function call for setting this value is Qt internal (not exposed to the outside).
What do you think if I use `QSettings` to read/write `qt.conf`?